int32 frameID
int32 syntime
int32 navID
uint8[2000] gridMsk
int32[1000] pObs
int32 nObs
